### ISSUE TITLE
Add popup information on agent located on the grid

### ISSUE THREAD

**Issue by hernanmd** [cormas/cormas#504]
For complex entities, some details could be displayed when the mouse is over the entity in the grid.
Add a method which answer a String with the desired detailed information about the entity without the need to open an inspector

**Comment by pierbo** [comment]
Yes, as it is the case for the cells (through the `interestingDetails` method), it could be good to have a small description of an agent when the mouse is over it.

BTW, `interestingDetails` must be corrected for SpatialEntity by changing `'Occupants: ' -> (self theOccupants size)` by `'Occupants: ' -> (self allOccupants size)`, as following:

```
interestingDetails
 	" Answer a <Collection> of interesting receiver's attributes, from a modeler point of view, to display in a popup for example "
 
 	^ super interestingDetails , { 
 		'Occupants: ' -> (self allOccupants size) . 
 		'Neighbours: ' -> (self neighbourhood size)
 		}
```
